FinniversKit/Sources/Assets/RadiobuttonAnimation.xcassets/radiobutton-select-0.imageset/Contents.json

```diff
@@ -9,13 +9,13 @@
       "scale" : "2x"
     },
     {
-      "idiom" : "universal",
       "filename" : "radiobutton-select-0@3x.png",
+      "idiom" : "universal",
```

Comment:
Non of these imagesets have any changes. Played around with rendering them as templates instead of default images so thats why all this json was flipped. Can just be ignored